controller:
  hero_chassis_controller:
    type: hero_chassis_controller/HeroChassisController
    wheel_track: 0.4
    wheel_base: 0.4
    wheel_radius: 0.07625
    #设置坐标系模式
    odomMode: true
    #功率限制值
    power_limit: 100.0
    power:
      effort_coeff: 12.0
      velocity_coeff: 0.0048
      power_offset: 0

  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
